postgresql_audit/flask.py

```diff
@@ -6,17 +6,40 @@
 from flask import g, request
 from flask.globals import _app_ctx_stack, _request_ctx_stack
 
-from .base import VersioningManager
+from .base import VersioningManager as BaseVersioningManager
 
 
-class FlaskVersioningManager(VersioningManager):
-    @property
-    def transaction_values(self):
+class VersioningManager(BaseVersioningManager):
```

Comment:
I was thinking it was more clean as it's in another module. But I can change back if you'd like?